packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -433,6 +433,40 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, MultiCaller
         return numerator.div(FixedPoint.Unsigned(totalSupply())).rawValue;
     }
 
+    /**
+     * @notice Computes the current liquidity utilization ratio.
+     * @dev Used in computing realizedLpFeePct off-chain.
+     */
+    function liquidityUtilizationCurrent() public returns (uint256) {
+        return liquidityUtilizationPostRelay(0);
+    }
+
+    /**
+     * @notice Computes the liquidity utilization ratio post a relay of known size.
+     * @dev Used in computing realizedLpFeePct off-chain.
+     * @param relayedAmount Size of the relayed deposit to factor into the utilization calculation.
+     */
+    function liquidityUtilizationPostRelay(uint256 relayedAmount) public returns (uint256) {
```

Comment:
we need to call `sync` first. so in the case that you are relay and then utilization variables that would work but it would not work if you did not sync within the same call. it would be possible to see the impact of a sync call, then compute the relative values of `pendingReserves`, `relayedAmount`, `utilizedReserves` but it seems much easier to do it this way with a method in the contract.